ui/plugin-system/src/runtime/time-range.ts

```diff
@@ -12,11 +12,13 @@
 // limitations under the License.
 
 import { createContext, useContext } from 'react';
-import { AbsoluteTimeRange, DurationString } from '@perses-dev/core';
+import { AbsoluteTimeRange, TimeRangeValue } from '@perses-dev/core';
 
 export interface TimeRange {
-  timeRange: AbsoluteTimeRange;
-  defaultDuration: DurationString;
+  timeRange: TimeRangeValue;
+  absoluteTimeRange: AbsoluteTimeRange;
```

Comment:
Do you think there's a use case for needing access to both of these? Or do consumers only care about the "current value" in absolute terms?